zcash_client_sqlite/src/wallet.rs

```diff
@@ -971,7 +969,7 @@ fn subtree_scan_progress<P: consensus::Parameters>(
                     OR subtree_end_height IS NULL",
                 ),
                 named_params! {
-                    ":start_height": u32::from(recover_until_height.unwrap_or(birthday_height) + 1),
+                    ":start_height": u32::from(recover_until_height.unwrap_or(birthday_height)),
```

Comment:
After this change, `WHERE subtree_end_height >= :start_height` would be more correct.